pilot/pkg/model/config.go

```diff
@@ -45,6 +45,40 @@ const (
 	RevisionLabel = "istio.io/rev"
 )
 
+// ConfigKey describe a specific config item depends on or get changed.
```

Comment:
ConfigKey describes the key of the config which changed

Reply:
It's not only used in `configsUpdated` but also `configDependencies`. I think it indicates more than "change".